exporter/awskinesisexporter/config.go

```diff
@@ -43,6 +43,7 @@ type Config struct {
 	AWS                AWSConfig `mapstructure:"aws"`
 	MaxRecordsPerBatch int       `mapstructure:"max_records_per_batch"`
 	MaxRecordSize      int       `mapstructure:"max_record_size"`
+	UseKplAggregation  bool      `mapstructure:"use_kpl_aggregation"`
```

Comment:
If we add this to the configuration, we'll need to improve the README for the component to at least reference an AWS doc saying what is Aggregation and potentially what improvements it can have.

Reply:
I think i also prefer `EnableAggregation` as oppose to `UseKPLAggregation` since it is concise and doesn't overload the user with new terms.